### DISCUSSION TITLE
Can we install CloudStack on Oracle linux 9?

### DISCUSSION THREAD

**Discussion by abdullahqureshi1994** [apache/cloudstack#9811] [Q&A] [answered]
I just want clarification that cloud stack can be installed on oracle linux or not?

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by kiranchavala** [comment, marked as answer]
@abdullahqureshi1994 yes you can use oracle linux 

https://docs.cloudstack.apache.org/en/4.19.1.1/releasenotes/compat.html#supported-os-versions-for-management-server

**Comment by abdullahqureshi1994** [comment]
Thanks @kiranchavala

**Comment by weizhouapache** [comment]
+1

it is also supported by smoke tests
![image](https://github.com/user-attachments/assets/a3013c33-99a0-47dc-b89d-7ddb8459ee5c)
